pages/404.js

```diff
@@ -0,0 +1,132 @@
+import Link from 'next/link'
+import css from "styled-jsx/css";
+
+export default function FourOhFour() {
+  return <>
+    <div className="header">
+        <div className="container">
+          <h1 className="title">Ooops, nothing to see around here…</h1>
+        </div>
+      </div>
+      <div className="content">
+        <div className="container">
+          <div className="card">
+            <div className="cardBody">
+            <button className="button">
+                <Link href="/"><a>Go back home</a></Link>
+            </button>
+            </div>
+          </div>
+        </div>
+      </div> 
+
+    <style global jsx>
+        {GlobalStyles}
+    </style>
+    <style jsx>{PageStyles}</style>
+  </>
+}
+
+
+const GlobalStyles = css.global`
```

Comment:
Should we import these from the page-renderer?

Reply:
Yeah it's not great to copy-paste styles. I have no idea how to do that.
@spbyrne how should we proceed here?
https://github.com/vercel/styled-jsx#global-styles